### PR TITLE
SDK-1853: Add github workflows for tests and sonar

### PR DESCRIPTION
**Pull request by emmas-yoti**

### CODE REVIEW

**Review by echarrod — CHANGES_REQUESTED** [review]
Please could we add a compatibility check for Python 3.9 now as well. 

I also don't think it gives us much benefit running the checks on the non-newest dev version:
image
(from https://python.domainunion.de/downloads/)
These older versions will only have security releases, so I think we're still safe to just run against dev in the latest two, maybe even just the latest one

**Review by gautam-yoti — APPROVED** [review]
Haven't worked on workflow setup, but LGTM 👍

**Review by echarrod — APPROVED** [review]
Looks good now :ok_hand: nice work Emma

**Comment by echarrod** [comment]
Just remembered, we'll need to update the Travis badge in the README too

**Review by echarrod — CHANGES_REQUESTED** [review]
Badge needs to be updated from Travis to Github too :+1:

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Yoti Python SDK #
 
-[![Build Status](https://travis-ci.com/getyoti/yoti-python-sdk.svg?branch=master)](https://travis-ci.com/getyoti/yoti-python-sdk)
+[![Build Status](https://github.com/getyoti/yoti-python-sdk/workflows/Unit%20Test/badge.svg?branch=master)](https://github.com/getyoti/yoti-python-sdk/actions)
```
Comment (echarrod) [inline README.md:3]: You can test these out with the current branch to check they work, before we merge into development (I checked <https://github.com/getyoti/yoti-python-sdk/workflows/Unit%20Test/badge.svg?branch=sdk-1853-github-actions>):
```suggestion
[![Build Status](https://github.com/getyoti/yoti-python-sdk/workflows/Unit%20Tests/badge.svg?branch=master)](https://github.com/getyoti/yoti-python-sdk/actions)
```

**Review by echarrod — CHANGES_REQUESTED** [review]

**Review by echarrod — APPROVED** [review]
Looks good, we can check the coverage is working in a future PR where we have Python files

**Pull request merged**